languages/python/exercises/concept/strings/.docs/introduction.md

```diff
@@ -0,0 +1,3 @@
+# Introduction
+
+A `str` typed object in Python is an immutable sequence of Unicode code points.
```

Comment:
It might be useful to explain how Python expects users to manipulate strings. Is it though built-in methods, functions? See [this example](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/introduction.md).

Reply:
This does need clarification ... strings are manipulated through the use of `overloaded-operators`, `string-methods`, and `builtin-functions`, among others.

Reply:
Added a bit of context here.